examples/http-server/main_test.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"testing"
+)
+
+func corazaGetRequest(getPath string) (int, string) {
+	if err := setupCoraza(); err != nil {
+		panic(err)
+	}
+	testServer := httptest.NewServer(corazaRequestHandler(http.HandlerFunc(hello)))
+	defer testServer.Close()
+
+	// Performs a get request
+	resp, err := http.Get(testServer.URL + getPath)
+	if err != nil {
+		log.Fatalln(err)
+	}
```

Comment:
Remember to close the response body after a successful HTTP call, otherwise it will lead you to a memory leak. Have a look at https://stackoverflow.com/a/33238755